managed-signers/src/utils/generateSignature.ts

```diff
@@ -0,0 +1,51 @@
+import { mnemonicToAccount } from "viem/accounts";
+import { getFid } from "./getFid";
+
+export const generate_signature = async function (public_key: string) {
+  if (typeof process.env.FARCASTER_DEVELOPER_MNEMONIC === "undefined") {
+    throw new Error("FARCASTER_DEVELOPER_MNEMONIC is not defined");
+  }
+
+  const FARCASTER_DEVELOPER_MNEMONIC = process.env.FARCASTER_DEVELOPER_MNEMONIC;
+  const FID = await getFid();
+
+  // DO NOT CHANGE ANY VALUES IN THIS CONSTANT
+  const SIGNED_KEY_REQUEST_VALIDATOR_EIP_712_DOMAIN = {
```

Comment:
can you switch to using @farcaster/core for these constants instead of defining them directly in this file.